shared/src/components/exercise/intro.cjsx

```diff
@@ -35,7 +35,7 @@ ExerciseIntro =  React.createClass
         <span>{TITLES[stepGroup]}</span>
       </h1>
       {getIntroText[stepGroup](project)}
-      <BS.Button onClick={onContinue}>Continue</BS.Button>
+      <a className="btn continue" onClick={onContinue}>Continue</a>
```

Comment:
Ah, I recall now - when you use`BS.Button`, the only way to get a `a` tag is to add an `href`, but then the `onCancel` doesn't call `.prefentDefault` so that jumps to the top of page when clicked.

Rather than adding custom handlers and stuff, I figured it was simplest to just use `a` since that's what QA wants.